README.md

```diff
@@ -120,24 +133,156 @@ compression_ctrl.export_model("compressed_model.pb", save_format='frozen_graph')
+dataset_loader = torch.utils.data.DataLoader(val_dataset, batch_size=1)
+
+# Step 1: Initialize transformation function
+input_name = onnx_model.graph.input[0].name
+def transform_fn(data_item):
+    images, _ = data_item
+    return {input_name: images.numpy()}
+
+# Step 2: Initialize NNCF Dataset
+calibration_dataset = nncf.Dataset(dataset_loader, transform_fn)
+# Step 3: Run the quantization pipeline
+quantized_model = nncf.quantize(onnx_model, calibration_dataset)
+```
+
+</details>
+
+<details><summary><b>OpenVINO</b></summary>
+
+```python
+import nncf
+import openvino.runtime as ov
+import torch
+from torchvision import datasets
+
+# Instantiate your uncompressed model
+model = ov.Core().read_model('/model_path')
+# Provide validation part of the dataset to collect statistics needed for the compression algorithm
+val_dataset = datasets.ImageFolder("/path")
+dataset_loader = torch.utils.data.DataLoader(val_dataset, batch_size=1)
+
+# Step 1: Initialize transformation function
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+
+# Step 2: Initialize NNCF Dataset
+calibration_dataset = nncf.Dataset(dataset_loader, transform_fn)
+# Step 3: Run the quantization pipeline
+quantized_model = nncf.quantize(model, calibration_dataset)
+```
+
+</details>
 
 ## Model Compression Samples
 
 For a quicker start with NNCF-powered compression, you can also try the sample scripts, each of which provides a basic training pipeline for classification, semantic segmentation and object detection neural network training correspondingly.
 
 To run the samples please refer to the corresponding tutorials:
 
+### Compression-Aware Training Samples
 - PyTorch samples:
   - [Image Classification sample](examples/torch/classification/README.md)
   - [Object Detection sample](examples/torch/object_detection/README.md)
   - [Semantic Segmentation sample](examples/torch/semantic_segmentation/README.md)
 - TensorFlow samples:
-  - [Image Classification sample](examples/tensorflow/classification/README.md)
-  - [Object Detection sample](examples/tensorflow/object_detection/README.md)
-  - [Instance Segmentation sample](examples/tensorflow/segmentation/README.md)
+    - [Image Classification sample](examples/tensorflow/classification/README.md)
+    - [Object Detection sample](examples/tensorflow/object_detection/README.md)
+    - [Instance Segmentation sample](examples/tensorflow/segmentation/README.md)
+
+### Post-Training Quantization Samples
+
+- [PyTorch Post-Training Quantization sample](examples/post_training_quantization/torch/mobilenet_v2/README.md)
+- [TensorFlow Post-Training Quantization sample](examples/post_training_quantization/tensorflow/mobilenet_v2/README.md)
+- [ONNX Post-Training Quantization sample](examples/post_training_quantization/onnx/mobilenet_v2/README.md)
+- [OpenVINO Post-Training Quantization sample](examples/post_training_quantization/openvino/mobilenet_v2/README.md)
 
 ## Model Compression Notebooks 
```

Comment:
Could you revise Installation section? I mean the following items:
- `(Experimental) For installation of NNCF for ONNXRuntime-OpenVINO, run:` -> `For installation of NNCF for ONNX`
- + add pip install nncf[openvino]

Reply:
Done